resources/js/components/home-footer.js

```diff
@@ -234,7 +248,7 @@ class HomeFooter extends LitElement {
     }
 
     postAppDataToServer(appSlug) {
-        const url = magic_url('unhide')
```

Comment:
@Perumal605 , please use the magic_url helper, also, notice the route has changed. It is currently "unhide", not "un-hide-app"

Reply:
@incraigulous I have updated magic_url helper.